mava/evaluator.py

```diff
@@ -275,28 +290,27 @@ def evaluator_setup(
             vmapped_eval_network_apply_fn,
             config,
             scanned_rnn,
+            win_rate,
         )
         absolute_metric_evaluator = get_rnn_evaluator_fn(
             eval_env,
             vmapped_eval_network_apply_fn,
             config,
             scanned_rnn,
+            win_rate,
             10,
         )
     else:
         vmapped_eval_network_apply_fn = jax.vmap(
             network.apply,
             in_axes=(None, 0),
         )
-        evaluator = get_ff_evaluator_fn(
-            eval_env,
-            vmapped_eval_network_apply_fn,
-            config,
-        )
+        evaluator = get_ff_evaluator_fn(eval_env, vmapped_eval_network_apply_fn, config, win_rate)
```

Comment:
Perhaps same formatting as the others
```suggestion
        evaluator = get_ff_evaluator_fn(
            eval_env,
            vmapped_eval_network_apply_fn,
            config,
            win_rate,
        )
```